app/src/main/res/layout/onboarding_jump_back_in_cfr.xml

```diff
@@ -22,8 +22,9 @@
             android:layout_marginTop="16dp"
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="16dp"
-            android:maxWidth="200dp"
-            android:text="@string/onboarding_home_screen_jump_back_contextual_hint"
+            android:maxWidth="279dp"
```

Comment:
where is this number derived from?

Reply:
For the `maxWidth` and `maxHeight` values, my intention was to respect the [Figma values](https://www.figma.com/file/7J6gh5D8DbGTExxskKdn3T/Mobile-Onboarding?node-id=1950%3A64573).

Here are a couple of screenshots:
| Current | After |
| ------ | ----- |
|![before](https://user-images.githubusercontent.com/35462038/185481180-a745c43d-28f3-4b67-8652-1048138a8460.png)|![after](https://user-images.githubusercontent.com/35462038/185481216-258af849-f5f8-439e-ae89-869dede5d1be.png)|



Should we stick with the `maxWidth = 200dp` and `wrap_content` height instead?

Reply:
For the height, I'd normally lean towards `wrap_content` so text can expand on small devices or devices with larger fonts without being cut-off.

For the width, I'd double check with UX to see if this is supposed to be a fixed width or wrap.

Reply:
Tagging @Kate-Galetski to check whether the CFR should respect the [design](https://www.figma.com/file/7J6gh5D8DbGTExxskKdn3T/Mobile-Onboarding?node-id=1950%3A64573)'s width (see `After` picture in the previous comment) or the current CFR width.

Reply:
Hi @Alexandru2909 the width looks fine to me! (atm we don't have any strict rules to how much % of the page the CFR should expand to.) What doesn't look fine to me is that this CFR is glued to the left border of the screen, while it actually should have a padding of 12px in there. [Link to figma](https://www.figma.com/file/d7l1iGabQYqnLcnQbHDGzM/Wallpaper-2.0-%2B-Onboarding-v106?node-id=5816%3A214700)
@gabrielluong do we need a separate ticket for that adjustment as well?

Reply:
This seems like a minor change so I added it in this ticket. In the figma designs, Android code tool indicates the value as `dp`, so here is an updated screenshot:
![Screenshot_20220824_223841](https://user-images.githubusercontent.com/35462038/186508507-e3130635-b2b7-4263-9b0e-e0b457ebc3a0.png)

If this should be addressed in another ticket we could also maintain the current `maxWidth` and update it in the CFR design issue.

Reply:
I am accepting these values since Kate has ok'd them. However, I would ask that you test this with a different language (say German) to see if we need any further followup.

Reply:
I tested with multiple languages and the CFR looks fine, apart from the rtl languages, as they use the `end` padding and make the CFR look glued to the left edge of the screen. For this I added the `12dp` padding to the end too.